data-types/collections/hashes.rb

```diff
@@ -43,9 +45,13 @@ def test_4
   end
 
   def test_5
-    # In the line below, create a new hash with
-    # a default value of zero
+    # In the line below, define the ingredients variable as a hash w/ the
+    # first key of "tomatoes" with a value of 0 and the second key of
+    # :carrots with a value of 0
```

Comment:
One thing to add here is that we still want this hash to be created with a default value.

Reply:
Hi Tim, i kept the note about defaulting the hash values to 0